invesalius/data/slice_.py

```diff
@@ -91,9 +91,9 @@ class Slice(metaclass=utils.Singleton):
     def __init__(self):
         self.current_mask = None
         self.blend_filter = None
```

Comment:
I think this is is not used anymore

Reply:
But `self.blend_filter` is used in `Slice.GetOutPut()` which is called in `invesalius/data/viewer_slice.py` in line 306.

Reply:
So left this way. But I'm almost sure this is not used anymore.